addons/account_payment/views/account_portal_templates.xml

```diff
@@ -55,19 +55,17 @@
     <template id="portal_invoice_page_inherit_payment" name="Payment on My Invoices" inherit_id="account.portal_invoice_page">
         <xpath expr="//t[@t-call='portal.portal_record_sidebar']//h2[1]" position="after">
             <t t-set="pending_txs" t-value="invoice.transaction_ids.filtered(lambda tx: tx.state in ('pending', 'authorized') and tx.provider_code not in ('none', 'custom'))"/>
-            <div class="d-grid">
-                <div t-if="invoice.payment_state in ('paid', 'in_payment')" class="position-sm-absolute position-lg-relative me-auto me-lg-0 top-0 end-0 rounded text-bg-success fs-6 fw-normal badge" >
-                    <i class="fa fa-fw fa-check-circle"/> Paid
-                </div>
-                <div t-elif="pending_txs" class="position-sm-absolute position-lg-relative me-auto me-lg-0 top-0 end-0 rounded text-bg-info fw-normal fs-6 badge" >
-                    <i class="fa fa-fw fa-check-circle"/> Pending
-                </div>
+            <div t-if="invoice.payment_state in ('paid', 'in_payment')" class="rounded text-bg-success fw-normal badge" >
+                <i class="fa fa-fw fa-check-circle"/> Paid
+            </div>
+            <div t-elif="pending_txs" class="position-sm-absolute position-lg-relative me-auto me-lg-0 top-0 end-0 rounded text-bg-info fw-normal fs-6 badge" >
```

Comment:
We remove a lot of classes from the `Paid` badge but we keep them all here. Is it normal ? They should be the same right ? I would expect this one to have (almost) the exact same classes.

Reply:
Indeed, this one should align with others and use only `rounded text-bg-info fw-normal badge`